core-library/src/main/java/org/silverpeas/core/util/logging/LogsAccessor.java

```diff
@@ -77,12 +79,13 @@ protected LogsAccessor() {
    * @param recordCount the number of records to get. O or a negative value means all the records.
    * @return an array of the last log records in the log at the time it was accessed.
    * @throws IOException if either the specified log doesn't exist or an error occurred wile
-   * getting the log records.
+   * @throws RelativeFileAccessException if the log name contains relative path.
    */
-  public String[] getLastLogRecords(String log, int recordCount) throws IOException {
+  public List<String> getLastLogRecords(String log, int recordCount)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, org.silverpeas.core.exception.RelativeFileAccessException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1160)